src/runtime/image.ts

```diff
@@ -5,16 +5,15 @@ import { imageMeta } from './utils/meta'
 import { checkDensities, parseDensities, parseSize, parseSizes } from './utils'
 import { prerenderStaticImages } from './utils/prerender'
 
-export function createImage(globalOptions: CreateImageOptions) {
+export function createImage(globalOptions: CreateImageOptions, isCalledInNitro: boolean = false) {
   const ctx: ImageCTX = {
     options: globalOptions,
   }
 
   const getImage: $Img['getImage'] = (input: string, options = {}) => {
     const image = resolveImage(ctx, input, options)
-
     // Prerender static images
-    if (import.meta.server && import.meta.prerender) {
+    if (import.meta.server && import.meta.prerender && !isCalledInNitro) {
```

Comment:
why do we not want to hint to prerender static images if called from a server endpoint?

Reply:
Thanks for the hint, i have tried the generate command and i see that beside this, i have a lot of unhandled situations!
I investigate the source code more precisely, review my code and come back with a solution ASAP.
Is there anything specific i need to know? like about writing tests or anything else?
Thanks for your time.